bin/glow-monitor/main.go

```diff
@@ -4,23 +4,47 @@ import (
 	"fmt"
 	"os"
 	"os/signal"
+	"path/filepath"
 	"syscall"
 
 	"github.com/glowlabs-org/gca-backend/client"
 )
 
 func main() {
 	// Create a new client, using the current directory as the basedir.
-	c, err := client.NewClient("/opt/glow-monitor/")
+	baseDir := "/opt/glow-monitor/"
+	c, err := client.NewClient(baseDir)
 	if err != nil {
 		fmt.Println("unable to create client: ", err)
 		return
 	}
 
 	// Wait for a shutdown signal from the OS.
 	sigChan := make(chan os.Signal, 1)
-	signal.Notify(sigChan, syscall.SIGINT, syscall.SIGTERM)
-	<-sigChan
+	signal.Notify(sigChan, syscall.SIGINT, syscall.SIGTERM, syscall.SIGUSR1, syscall.SIGUSR2)
+
+	done := false
+	for !done {
+		s := <-sigChan
+		switch s {
+		case syscall.SIGINT:
+			done = true
+		case syscall.SIGTERM:
+			done = true
+		case syscall.SIGUSR1:
+			// Dump status to the terminal.
+			fmt.Printf("%v", c.DumpEventLogs())
+		case syscall.SIGUSR2:
+			// Write the status to a file "status.txt" in the client directory. File will be
+			// created and/or truncated before writing.
+			path := filepath.Join(baseDir, "status.txt")
```

Comment:
probably better to use `c.baseDir` here

Reply:
it's the same value but  this way the code is more easily shuffled around

Reply:
c.staticBaseDir is not available from the main package